dotCMS/src/main/java/com/dotcms/browser/BrowserAPIImpl.java

```diff
@@ -268,25 +276,42 @@ private Tuple2<String, List<Object>> selectQuery(final BrowserQuery browserQuery
         final StringBuilder sqlQuery = new StringBuilder("select cvi." + workingLiveInode + " as inode "
                 + " from contentlet_version_info cvi, identifier id, structure struc, contentlet c "
                 + " where cvi.identifier = id.id and struc.velocity_var_name = id.asset_subtype and  "
-                + " c.inode = cvi." + workingLiveInode);
+                + " c.inode = cvi." + workingLiveInode + " and cvi.variant_id='"+DEFAULT_VARIANT.name()+"' ");
```

Comment:
I might be wrong, but the BrowserAPITest doesn't seem to be covering all the scenarios implemented on this logic where you force the ES search if the record doesn't exist in database